app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -28,6 +27,11 @@ sealed class RouteAction : Action {
     ) : RouteAction() {
         class SecurityDisclaimerDialog(setUpAction: Action) : DialogAction(positiveButtonAction = setUpAction)
     }
+
+    sealed class DialogFragment(val dialogTitle: String, val dialogSubtitle: String? = null) : RouteAction() {
```

Comment:
Ooo, I like this abstraction!

Reply:
Nit: use `@StringRes Int`s instead of `String`s, unless we are doing any string interpolation.